bin.js

```diff
@@ -44,16 +44,17 @@ if (napi.isNapiRuntime(rc.runtime)) napi.logUnsupportedVersion(rc.target, log)
 
 var pm = whichPmRuns()
 var isNpm = !pm || pm.name === 'npm'
+var origin = util.packageOrigin(process.env, pkg)
 
 if (!isNpm && /node_modules/.test(process.cwd())) {
   // From yarn repository
 } else if (opts.force) {
   log.warn('install', 'prebuilt binaries enforced with --force!')
   log.warn('install', 'prebuilt binaries may be out of date!')
-} else if (!(typeof pkg._from === 'string')) {
+} else if (!origin) {
   log.info('install', 'installing standalone, skipping download.')
   process.exit(1)
```

Comment:
I'm leaning towards removing this check altogether, because a user (and build tools that wrap `prebuild-install`) can do `npm i --build-from-source`. More importantly, I prefer erroneously downloading over erroneously building, because downloading does not require a build toolchain and this benefits way more users than the few that want to build.